### DISCUSSION TITLE
Is Github Actions Free for ALL os's

### DISCUSSION THREAD

**Discussion by ccocks** [community/community#136961] [Actions] [answered]
### Select Topic Area

Question

### Body

Is Github Actions Free for ALL os's?
I know that you can use unlimited minutes per month for public repos, but is it unlimited only for `ubuntu-latest` or will any OS work for free (e.g. `windows-latest`, 'macos-latest`)

**Comment by theshawnv** [comment, marked as answer]
GitHub Actions is free for public repositories on all operating systems (Linux, macOS, and Windows). For private repositories, GitHub Actions provides a certain amount of free usage per month depending on the plan you are on.
you can fallow the pricing on this link. [https://docs.github.com/en/billing/managing-billing-for-github-actions/about-billing-for-github-actions](url)

**Reply by ccocks** [reply]
Thank you for the clarification.
